packages/util-cli-reporter/__tests__/unit/reporter.test.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * __tests__/unit/reporter.js
+ * Test: lib/reporter.js
+ */
+'use strict';
+
+const stripAnsi = require('strip-ansi');
+const reporter = require('../../lib/reporter');
+
+describe('Reporter Basics', () => {
+	test('exports `reporter` as an object', () => {
+		expect(reporter).toBeTruthy();
+		expect(typeof reporter).toBe('object');
+	});
+
+	test('has a `info` method', () => {
```

Comment:
are these "has a [value] method" tests worthwhile? is it good practice to go through each public method of something and write a basic unit test to say it exists? I'm asking more so I learn something than anything else

Reply:
I wouldn't say they were massively important, but it does validate the way the module is structured